src/app/settings/views/Configuration/CommissioningForm/CommissioningForm.test.tsx

```diff
@@ -77,13 +77,22 @@ describe("CommissioningForm", () => {
         </MemoryRouter>
       </Provider>
     );
-    submitFormikForm(wrapper, {
-      commissioning_distro_series: "bionic",
-      default_min_hwe_kernel: "ga-16.04-lowlatency",
-      maas_auto_ipmi_user: "maas",
-      maas_auto_ipmi_k_g_bmc_key: "password",
-      maas_auto_ipmi_user_privilege_level: "USER",
+
+    const maas_auto_ipmi_user_input = screen.getByRole("textbox", {
+      name: "MAAS generated IPMI username",
     });
+    await userEvent.clear(maas_auto_ipmi_user_input);
+    await userEvent.type(maas_auto_ipmi_user_input, "maas");
+
+    const maas_auto_ipmi_k_g_bmc_key_input = screen.getByLabelText(
+      "K_g BMC key textbox"
+    );
+    await userEvent.clear(maas_auto_ipmi_k_g_bmc_key_input);
+    await userEvent.type(maas_auto_ipmi_k_g_bmc_key_input, "password");
+
+    await userEvent.click(screen.getByRole("radio", { name: "User" }));
+
+    await userEvent.click(screen.getByRole("button", { name: "Save" }));
```

Comment:
This could use an enum for the label.

Reply:
I'm not certain about that for the "Save" button - currently that label is defined in `src/app/base/components/FormikFormButtons/FormikFormButtons.tsx`, would it be worth adding enums for the labels there? Would probably need to go through other migrated tests and import these too

Reply:
It'd be fine to add those labels in `FormikFormButtons`, that would make it easier to update in the future if we wanted the default to be something other than "Save".